lib/linguist/languages.yml

```diff
@@ -5014,7 +5014,7 @@ WebIDL:
   codemirror_mode: webidl
   codemirror_mime_type: text/x-webidl
   language_id: 395
-Windows Registry:
+Windows Registration Entries:
```

Comment:
I'd say that `Windows Registry Entry` is more accurate, actually. I've never heard the word "Registration" used in the context of the Windows Registry...

Reply:
https://support.microsoft.com/en-us/help/310516/how-to-add-modify-or-delete-registry-subkeys-and-values-by-using-a-reg

Reply:
Yes, but... calling it `Windows Registration` sounds rather misleading...